File: DP_FamilyPhotoSharingiOS-main/DP_FamilyPhotoSharingiOS-main/DP_iOSApp/FamilyPhotoSharing/FamilyPhotoSharing/RequestModel/LoadPhotosRequest.swift
//
//  LoadPhotosRequest.swift
//  FamilyPhotoSharing
//
//  Created by Jiří Gába on 01.05.2026.
//

struct LoadPhotosRequest: Codable {
    let thumbnailData: Int
    let page: Int
    let part: Int
    let albumId: Int

    enum CodingKeys: String, CodingKey {
        case thumbnailData = "ThumbnailData"
        case page = "Page"
        case part = "Part"
        case albumId = "AlbumId"
    }
}

File: DP_FamilyPhotoSharing-main/DP_FamilyPhotoSharing-main/FamilyPhotoSharing/ApiRequests/LoadPhotosRequest.cs
namespace FamilyPhotoSharing.ApiRequests
{
    public class LoadPhotosRequest
    {
        public int Page { get; set; }
        public int Part { get; set; }
        public int ThumbnailData { get; set; }
        public int AlbumId { get; set; }
    }
}
